src/azureFunction/azureFunctionProjectService.ts

```diff
@@ -0,0 +1,186 @@
+		const defaultLineSet = new Set(defaultSqlBindingTextLines);
+		let replacedValueLines = [];
+		for (let defaultLine of newValueLines) {
+			// Skipped lines
+			if (defaultLineSet.has(defaultLine.trimStart())) {
+				continue;
+			} else if (defaultLine.trimStart() === defaultBindingResult) { // Result change
+				replacedValueLines.push(defaultLine.replace(defaultBindingResult, sqlBindingResult));
+			} else {
+				// Normal lines to be included
+				replacedValueLines.push(defaultLine);
+			}
+		}
+		defaultBindedFunctionText = replacedValueLines.join(os.EOL);
+		fs.writeFileSync(filePath, defaultBindedFunctionText, 'utf-8');
+	}
+}
+
+/**
+ * A C# Azure Functions project must be present in order to create a new Azure Function for the table
+ * @returns the selected project file path or indicates to user there is no C# Azure Functions project
+ */
+async function isAzureFunctionProjectOpen(): Promise<string> {
+	let selectedProjectFile: string | undefined = '';
+	if (vscode.workspace.workspaceFolders === undefined || vscode.workspace.workspaceFolders.length === 0) {
+		return selectedProjectFile;
+	} else {
+		const hostFiles = await getHostFile();
+		const projectFiles = await getProjectFile();
+		if (projectFiles !== undefined && hostFiles !== undefined) {
+			// Check to see if its only a function project using host.json and project file
+			let functionProjects = [];
+			let count = 0;
+			for (let host of hostFiles) {
+				for (let project of projectFiles) {
+					path.dirname(host) === path.dirname(project) ? functionProjects.push(project) && count++ : count;
+				}
+			}
+			if (count > 1) {
+				// select project to add azure function to
+				selectedProjectFile = (await vscode.window.showQuickPick(projectFiles, {
+					canPickMany: false,
+					title: constants.selectProject,
+					ignoreFocusOut: true
+				}));
+				return selectedProjectFile;
+			} else if (count === 1) {
+				// only one azure function project found
+				return functionProjects[0];
+			}
+		}
+		return;
+	}
+}
+
+// gets the azure functions project file path
+async function getProjectFile(): Promise<string[] | undefined> {
+	const projUris = await vscode.workspace.findFiles('**/*.csproj');
+	let projFiles: string[] = [];
+	for (let projUri of projUris) {
+		projFiles.push(projUri.fsPath);
+	}
+	return projFiles.length > 0 ? projFiles : undefined;
+}
+
+// gets the host file config file path
+async function getHostFile(): Promise<string[] | undefined> {
+	const hostUris = await vscode.workspace.findFiles('**/host.json');
+	let hostFiles = [];
+	for (let hostUri of hostUris) {
+		hostFiles.push(hostUri.fsPath);
+	}
+	return hostFiles.length > 0 ? hostFiles : undefined;
+}
+
+// gets the local.settings.json file path
+async function getSettingsFile(): Promise<string | undefined> {
+	const settingsFiles = await vscode.workspace.findFiles('**/local.settings.json');
+	return settingsFiles.length > 0 ? settingsFiles[0].fsPath : undefined;
```

Comment:
This is hardcoded as well to just use the first local.settings.json that it finds which isn't correct (especially now that we're picking the project from the list).